### PR TITLE
Apply new rules for determining case responsibility

### PR DESCRIPTION
**Pull request by TobyRet**
Rules applied as illustrated.

Case_responsibility_decision_tree

### CODE REVIEW

File: app/services/responsibility_service.rb [hunk 1]
```diff
@@ -1,30 +1,92 @@
...
     else
-      'Unknown'
+      UNKNOWN
     end
   end
 
   def self.calculate_pom_responsibility(offender)
-    return 'Unknown' if offender.release_date.nil?
-    return 'Responsible' if assign_responsible?(offender)
+    return UNKNOWN if offender.release_date.nil?
```
Comment (TobyRet) [inline app/services/responsibility_service.rb]: I've introduced guard clauses in order to remove if / else statements.
I'm not sure if iterating over rules will bring about much benefit, if anything might make the card harder to read? Welcome opinions!

Comment (TobyRet) [reply app/services/responsibility_service.rb]: Also welcome suggestions regarding code climate issues (too may return methods). The class method could be split but then I'm not sure if this then makes the code more readable?

Comment (rossjones) [reply app/services/responsibility_service.rb]: Looks already and is pretty clear.  Might be tempted to pre-calculate the values to save calling functions multiple times, and I don't think welsh? is required after the first guard?

Comment (TobyRet) [reply app/services/responsibility_service.rb]: Thanks for the comments - I've pushed the changes :-)

**Pull request merged**